### ISSUE TITLE
NTR: Renal biopsy: lymphangiectasia (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9363]
New term request Renal biopsy: lymphangiectasia (elthat_141010044002): Kidney  UMLS: 1856132

**Comment by pnrobinson** [comment]
@avizrosenberg @yongqunh @MickeySegal
Avi, could you please check this term request?

def:A developmental anomaly of renal lymphatics that results from a failure of the lymphatics to make communication with the rest of the lymph trunks. The abnormal lymphatics thus dilate and form cystic spaces at the renal sinus and
in the perinephric space.

syn: Renal lymphangiomatosis, Peripelvic lymphangiectasia, Hygroma renale, Polycystic disease of the renal sinus

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000073
- label: Renal lymphangiectasia
- parents: [Abnormal renal morphology (HP:0012210)](https://hpo.jax.org/app/browse/term/HP:0012210)
- pmid: PMID:17684071
- def: A developmental anomaly of renal lymphatics that results from a failure of the lymphatics to make communication with the rest of the lymph trunks. The abnormal lymphatics thus dilate and form cystic spaces at the renal sinus and in the perinephric space.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9363
- synonyms: Hygroma renale(NONE)|Renal lymphangiomatosis(NONE)|Peripelvic lymphangiectasia(NONE)
